modules/brew/brew-bash-completions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# shellcheck shell=sh disable=SC1091,SC2039,SC2166
```

Comment:
This line seems to set the shell to `sh` explicitly? @mecattaf is this intentional? Also see my other comment https://github.com/blue-build/modules/pull/239/files#r1622418859

```suggestion
# shellcheck shell=bash disable=SC1091,SC2166	
```

https://github.com/koalaman/shellcheck/wiki/SC2039 not necessary to ignore if using actually bash (which we are, these are bash completions and contain multiple bashisms)

Reply:
@xynydev `sh` in Fedora is a symlink to `bash`. So this is effectively `bash`.

But I agree that setting it explicitely to `bash` is better.